v2g_liberty.py

```diff
@@ -317,15 +319,17 @@ def set_next_action(self):
                 # Intended for the situation where the car returns from a trip with a low battery.
                 # An SoC below the minimum SoC is considered "unhealthy" for the battery,
                 # this is why the battery should be charged to this minimum asap.
-
-                self.log(f"Starting max charge now and not requesting schedule based on SoC below"
-                         f" minimum ({self.CAR_MIN_SOC_IN_PERCENT}%).")
+                message = f"Car battery state of charge ({self.connected_car_soc}%) is too low. " \
+                          f"Charging with maximum power until minimum of ({self.CAR_MIN_SOC_IN_PERCENT}%) is reached."
+                self.log(message)
```

Comment:
Alternatively, log every user notification message by default?

For example, by including a flag like this: `def notify_user(..., also_log_message: bool = True)`

Reply:
Good suggestion, will do for now always log message.